server/models/executor.js

```diff
@@ -0,0 +1,30 @@
+var async = require('async');
+
+module.exports = function(Executor) {
+  Executor.observe('after save', function(ctx, next) {
+    var serviceManager = Executor.app.serviceManager;
+    if (ctx.instance) {
+      if (serviceManager.onExecutorUpdate.length === 2) {
```

Comment:
Not necessary here, since this handler is being defined for the first time in this PR.